pkg/timeutil/timeutil.go

```diff
@@ -26,3 +42,22 @@ func LastWeekday(weekday time.Weekday) time.Time {
 
 	return date
 }
+
+// StringToWeekday maps a string to a time.Weekday.
+func StringToWeekday(day string) (time.Weekday, error) {
+	timeWeekday, ok := weekday[strings.ToLower(day)]
+	if !ok {
+		return time.Sunday, fmt.Errorf("timeutil: Invalid weekday '%s' given", day)
+	}
+
+	return timeWeekday, nil
+}
+
+// FormatDuration prints the time using the given format.
+func FormatDuration(duration time.Duration, timeFormat string) string {
+	if timeFormat == "decimal" {
```

Comment:
Hmm, the actual type itself is a string though, the config value is a string really and it'll probably confuse the TOML library if you set the type to something like that. 

The real reason behind this is so that you're not comparing against a string when you need lookup a day (it's helpful if it's referenced many times in the app to ensure consistency).